README.md

```diff
@@ -6,17 +6,16 @@
 # Compiler Explorer
 
 **Compiler Explorer** is an interactive compiler exploration website. Edit code in C, C++, C#, F#, Rust, Go, D, Haskell,
-Swift, Pascal, [ispc](https://ispc.github.io/), Python, Java or in any of the other
-[30+ supported languages](https://godbolt.org/api/languages), and see how that code looks after being compiled in real
-time. Multiple compilers are supported for each language, many different tools and visualisations are available, and the
+Swift, Pascal, [ispc](https://ispc.github.io/), Python, Java, or any of the other
+[30+ supported languages](https://godbolt.org/api/languages), and see how that code looks after being compiled in real time. Multiple compilers are supported for each language, many different tools and visualizations are available, and the
 UI layout is configurable (thanks to [GoldenLayout](https://www.golden-layout.com/)).
 
 Try out at [godbolt.org](https://godbolt.org), or [run your own local instance](#running-a-local-instance).
 
 **Compiler Explorer** follows a [Code of Conduct](CODE_OF_CONDUCT.md) which aims to foster an open and welcoming
 environment.
 
-**Compiler Explorer** was started in 2012 to show how C++ constructs translated to assembly code. It started out as a
+**Compiler Explorer** was started in 2012 to show how C++ constructs are translated to assembly code. It started as a
```

Comment:
I don't know that this materially helps the understanding but if you feel this is simpler to understand...